exercise_03/index.js

```diff
@@ -1 +1,17 @@
-//import and execute
+// Import and execute
+import generateRandomNumberInRange from "./generateRandomNumberInRange.js";
+
+try {
```

Comment:
A/I smell...
* comments in Spanish
* use of `try/catch`, not yet explained.

- [ ] 🚨 Explain this implementation